x-pack/test/alerting_api_integration/spaces_only/tests/actions/enqueue.ts

```diff
@@ -70,63 +70,6 @@ export default function ({ getService }: FtrProviderContext) {
       await esTestIndexTool.waitForDocs('action:test.index-record', reference, 1);
     });
 
-    it('should cleanup task after a failure', async () => {
```

Comment:
I removed this test, because failures are retried now and test below tests this new change